hazelcast/src/main/java/com/hazelcast/config/SplitBrainProtectionConfig.java

```diff
@@ -71,7 +71,9 @@
     private String name;
     private boolean enabled;
     private int minimumClusterSize = 2;
-    private List<SplitBrainProtectionListenerConfig> listenerConfigs = new ArrayList<SplitBrainProtectionListenerConfig>();
+
```

Comment:
I see that the merge introduced some empty lines here. Will remove them in the next change.